api/controller/psmdb_cluster_api.proto

```diff
@@ -81,8 +81,19 @@ message UpdatePSMDBClusterRequest {
       string_not_empty: true
     }
   ];
-  // Cluster parameters.
-  PSMDBClusterParams params = 3 [
```

Comment:
let's keep `params`, by inline I meant put new `UpdatePSMDBClusterParams` inside `UpdatePSMDBClusterRequest` like `Cluster` in `CreatePSMDBClusterRequest`